tests/robustness/model/deterministic_test.go

```diff
@@ -20,378 +20,382 @@ import (
 	"go.etcd.io/etcd/api/v3/mvccpb"
 )
 
-func TestModelBase(t *testing.T) {
-	type testOperation struct {
-		req     EtcdRequest
-		resp    EtcdResponse
-		failure bool
+func TestModelDeterministic(t *testing.T) {
+	for _, tc := range deterministicModelTestScenarios {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
nit, and the same comment for the nonDeterministicTestScenarios.
```suggestion
		tc := tc
		t.Run(tc.name, func(t *testing.T) {
```

Reply:
Done